overview.rst

```diff
@@ -1,129 +1,316 @@
+   operativo ospite quale token crittografico esterno. Nel caso di
+   autenticazione effettuata tramite un dispositivo mobile, é possibile
+   scaricare gratuitamente e installare l'App «CieID» direttamente dallo
+   Store online
+   (`Android <https://play.google.com/store/apps/details?id=it.ipzs.cieid>`__
+   o `iOS <https://apps.apple.com/it/app/cieid/id1504644677>`__).
 
-   Schema di autenticazione Entra con CIE tramite modalitá "mobile".
+Allo stato dell'arte questa modalità è fruibile mediante smartphone
+dotati di tecnologia NFC e sistema operativo Android 6 o superiore,
+mediante il browser “Chrome”, e iPhone 7 o superiori con sistemi
+operativi iOS 13 o superiore [#]_, mediante browser Safari. Tutte le
+componenti software, sia lato server IdP e sia client (Middleware e App
+CieID), sono sviluppate e gestite dal Poligrafico che cura anche le
+attività di supporto e assistenza tecnica al Service Provider
+nell'utilizzo di tali strumenti e durante l'intero iter di integrazione
+dello schema «Entra con CIE» all'interno dei servizi erogati dai SP.
 
-.. figure:: ./media/img_MSC_CIE_ibrido.png
-   :alt: Modalitá computer + smartphone.
-   :width: 15 cm
-   :name: modalitá-computer-smartphone
+I diagrammi seguenti illustrano i meccanismi di funzionamento dello
+schema “Entra con CIE” nei vari scenari di utilizzo, secondo il
+protocollo impiegato e il livello di sicurezza richiesto.
 
-   Schema di autenticazione Entra con CIE tramite modalitá "computer + smartphone".
+.. [#]
+   Non è consentito l'accesso da terminali dotati di sistema operativo
+   iOS precedenti alla release 13 a causa dell'impossibilità di impiego
+   del lettore NFC per contesti di utilizzo non approvati da Apple.
 
-Lo schema Entra con CIE si realizza mediante due macro fasi distinte: 
+.. _sec-meccanismi:
 
-   1. richiesta del servizio esposto dal portale/app del Service Provider che avviene all'interno del browser dell'utente nel dominio del SP;
+Meccanismi di funzionamento
+---------------------------
 
-   2. autenticazione dell'utente effettuata direttamente dall'Identity Provider.
 
-Per quanto concerne il primo punto, la richiesta avviene tramite una "*call to action*" realizzata dal Service Provider tramite un apposito pulsante "Entra con CIE" e che ha come *landing page* un *endpoint* del Ministero dell'Interno il quale innesca il processo di identificazione mediante la componente server CieID dell'IdP. Per consentire una esperienza utente quanto più possibile omogenea presso tutti i service provider che integrano lo schema di identificazione mediante la CIE si deve utilizzare il kit disponibile all'indirizzo https://github.com/italia/cie-graphics.
+Accesso di livello 1
+~~~~~~~~~~~~~~~~~~~~
 
-.. figure:: ./media/pulsanteEntraConCIE.png
-   :alt: Pulsante Entra con CIE
-   :scale: 70 %
-   :name: pulsante-entra-con-cie
+**DESKTOP**
 
-   Pulsante ufficiale "Entra con CIE"
+.. figure:: media/image3.png
```

Comment:
@GMindMercatis @aantetomaso , mi pare di vedere che i due sequence diagram relativi al liv desktop o smartphone sono identici (e direi che ha senso dal mio punto di vista). Quindi suggerirei di eliminare uno dei due e dire che si riferiscono appunto a entrambi gli scenari. Che ne dite?

Reply:
Inoltre suggerirei l'utilizzo di https://sequencediagram.org/ così possiamo inserire poi il link alla preview e facilitare la lettura del diagramma (oltre al fatto che ha una qualità molto alta)

Reply:
Concordo nell'eliminazione del secondo diagramma (ho proceduto a modificare), meno sull'utilizzo dell'altro tool, come formalismo e resa preferisco Visio. 
Ad ogni modo, come preferite.

Reply:
Antonio non vedo ancora tue modifiche su questo file...non hai ancora fatto push?

Reply:
Non ancora, ho difficoltà in azienda. Cerco di farlo stasera.